app/services/integrations/aggregator/taxes/invoices/void_service.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+module Integrations
+  module Aggregator
+    module Taxes
+      module Invoices
+        class VoidService < BaseService
+          def action_path
+            "v1/#{provider}/void_invoices"
+          end
+
+          def call
+            return result unless integration
+            return result unless integration.type == 'Integrations::AnrokIntegration'
```

Comment:
I'm wondering, since we do this check in every service that is inherited from BaseService, can we move it to init in base?

Reply:
🤔 Probably raising an error in initailizer could work, even not my style preference, but returning result in initializer can affect how this object is called.
Probably if those checks list will grow, we can add one helper method in parent class:)